brianlong/www.validators.app
# 182816572_create_data_center_for_validators_without_ip

#### What's this PR do?
- Create script to append validators to data center with key Unknown when no ip exists

#### How should this be manually tested?
- Run: `d = DataCenter.create(traits_autonomous_system_number: 0, city_name: "Unknown")`, `d.data_center_hosts.create(host: nil)` (it is needed only for one time)
- Run script locally, you should see new Data Center "Unknown" with some validators assigned
- Check the log file

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/182816572)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
